pkgs/by-name/xd/xdg-desktop-portal-termfilechooser/package.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeWrapper
+, meson
+, ninja
+, pkg-config
+, wayland-protocols
+, wayland-scanner
+, inih
+, libdrm
+, mesa
+, scdoc
+, systemd
+, wayland
+}:
+
+stdenv.mkDerivation {
+  pname = "xdg-desktop-portal-termfilechooser";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "nani8ot";
```

Comment:
Why this fork specifically? Doesn't seem like they add anything useful. And it doesn't match the homepage either

Reply:
Looks like the original repo is dead. There are several forks but each one is doing their own thing: https://github.com/GermainZ/xdg-desktop-portal-termfilechooser/network

Reply:
@jtojnar The forks look dead too, which is probably find, if development is "done". Just wondering why this fork specifically, and IMO the homepage should match.